H4 fact.h3c scenarios 2-3-4-5 (#71)
nomises/Traduction-Hota-FR: Traduction/campagne_tempo_txt/H4Fact/0404.txt

Name:

> Frédéric rencontre Kastore

Reviewer: ```suggestion
Frederick rencontre Kastore
```

Message:

> Je me rendais à Volta, une ville libre située à l'est de Burton. Le récent tremblement de terre avait détruit les usines qui s'y trouvaient, mais j'espérais trouver des compagnons parmi ceux qui s'étaient retrouvés involontairement sans emploi. Cette ville était le dernier endroit où Henrietta et son groupe avaient été vus. Cet après-midi-là, j'ai embarqué sur un navire au port d'Orca's Maw : j'avais l'intention de contourner la chaîne de montagnes voisine par la mer et d'atteindre Volta, d'où partirait mon expédition aérienne.		Imaginez ma surprise lorsque, juste avant de lever l'ancre, un messager m'a rattrapé et m'a annoncé : « Kastore, le roi de Deyja, a l'intention de se rendre à Burton afin d'établir et de renforcer les liens économiques et souhaite rencontrer Frederick. »		Le roi de Deyja ? Ici ? Bien sûr, j'avais entendu parler de l'arrivée au pouvoir d'un certain Kastore l'année dernière dans ce pays de nécromanciens et de sorciers noirs, mais je ne m'étais jamais intéressé à ces querelles. Les coups d'État sont monnaie courante à Deyja. Mais que pouvait-il bien vouloir ici, et quels liens économiques pouvait-il y avoir alors qu'il n'y avait pas un seul port à Deyja ? Et comment connaissait-il mon nom ? D'un autre côté, le monde de la magie est petit, et celui des artisans encore plus. J'ai ordonné à un brick rapide de partir et de proposer au capitaine du navire de Kastore de faire escale à Volta plutôt qu'à La Gueule de l'Orque. Je manquais de temps, mais les préparatifs de l'expédition devaient encore durer quelques jours, et j'espérais que la rencontre avec cet invité inattendu ne prendrait pas trop de temps. Je ne voulais pas ignorer la visite du souverain de Deyja et me faire un nouvel ennemi — trop de sorciers puissants figuraient déjà sur la liste... et qui sait, peut-être que ce Kastore se révélerait être un interlocuteur intéressant ? J'ai été frappé par la vue qui s'offrait à nous alors que nous approchions du port de Volta. De loin, j'ai aperçu un grand navire aux voiles grises, capable de transporter facilement toute une armée. Quelle chance que nous ayons dragué le port l'année précédente afin que d'énormes cargos à fort tirant d'eau puissent y décharger leur cargaison. C'est alors que j'ai vu Kastore pour la première fois. Un elfe en armure brillante se tenait debout près de la passerelle du navire géant, les mains derrière le dos. De chaque côté de lui, des femmes serpents à plusieurs bras se tenaient immobiles comme des statues, armées jusqu'aux dents. Les nagas sont courants à Bracada, où ils sont au service de nombreux mages fortunés, mais je savais que tout sorcier habile pouvait invoquer ces créatures redoutables. Il était cependant plus difficile de leur faire obéir aux ordres, ce que seuls les sorciers les plus puissants pouvaient faire. Mon arrivée ne sembla pas impressionner outre mesure le seigneur de Deyja. Sans même tourner la tête, il dit d'un ton sec :			« Je m'attendais à vous voir, mais vous m'avez fait attendre... Frederick, c'est bien ça ? » « C'est exact, répondis-je. Que puis-je faire pour vous ? » Kastore me fit face tranquillement. « La nouvelle de vos inventions se répand dans tous les coins d'Enroth, et les plus sages y prêtent attention. Deyja ne fait bien sûr pas exception. Vous ne me connaissez que comme le souverain d'un pays lointain, mais j'aimerais que vous me considériez comme un collègue, Frederick. Je suis moi-même une sorte d'inventeur. »

Reviewer: Espacement
```suggestion
Je me rendais à Volta, une ville libre située à l'est de Burton. Le récent tremblement de terre avait détruit les usines qui s'y trouvaient, mais j'espérais trouver des compagnons parmi ceux qui s'étaient retrouvés involontairement sans emploi. Cette ville était le dernier endroit où Henrietta et son groupe avaient été vus. Cet après-midi-là, j'ai embarqué sur un navire au port d'Orca's Maw : j'avais l'intention de contourner la chaîne de montagnes voisine par la mer et d'atteindre Volta, d'où partirait mon expédition aérienne.		Imaginez ma surprise lorsque, juste avant de lever l'ancre, un messager m'a rattrapé et m'a annoncé : « Kastore, le roi de Deyja, a l'intention de se rendre à Burton afin d'établir et de renforcer les liens économiques et souhaite rencontrer Frederick. »		Le roi de Deyja ? Ici ? Bien sûr, j'avais entendu parler de l'arrivée au pouvoir d'un certain Kastore l'année dernière dans ce pays de nécromanciens et de sorciers noirs, mais je ne m'étais jamais intéressé à ces querelles. Les coups d'État sont monnaie courante à Deyja. Mais que pouvait-il bien vouloir ici, et quels liens économiques pouvait-il y avoir alors qu'il n'y avait pas un seul port à Deyja ? Et comment connaissait-il mon nom ? D'un autre côté, le monde de la magie est petit, et celui des artisans encore plus. J'ai ordonné à un brick rapide de partir et de proposer au capitaine du navire de Kastore de faire escale à Volta plutôt qu'à La Gueule de l'Orque. Je manquais de temps, mais les préparatifs de l'expédition devaient encore durer quelques jours, et j'espérais que la rencontre avec cet invité inattendu ne prendrait pas trop de temps. Je ne voulais pas ignorer la visite du souverain de Deyja et me faire un nouvel ennemi — trop de sorciers puissants figuraient déjà sur la liste... et qui sait, peut-être que ce Kastore se révélerait être un interlocuteur intéressant ?		J'ai été frappé par la vue qui s'offrait à nous alors que nous approchions du port de Volta. De loin, j'ai aperçu un grand navire aux voiles grises, capable de transporter facilement toute une armée. Quelle chance que nous ayons dragué le port l'année précédente afin que d'énormes cargos à fort tirant d'eau puissent y décharger leur cargaison. C'est alors que j'ai vu Kastore pour la première fois. Un elfe en armure brillante se tenait debout près de la passerelle du navire géant, les mains derrière le dos. De chaque côté de lui, des femmes serpents à plusieurs bras se tenaient immobiles comme des statues, armées jusqu'aux dents. Les nagas sont courants à Bracada, où ils sont au service de nombreux mages fortunés, mais je savais que tout sorcier habile pouvait invoquer ces créatures redoutables. Il était cependant plus difficile de leur faire obéir aux ordres, ce que seuls les sorciers les plus puissants pouvaient faire.		Mon arrivée ne sembla pas impressionner outre mesure le seigneur de Deyja. Sans même tourner la tête, il dit d'un ton sec :			« Je m'attendais à vous voir, mais vous m'avez fait attendre... Frederick, c'est bien ça ? »		« C'est exact, répondis-je. Que puis-je faire pour vous ? »		Kastore me fit face tranquillement.		« La nouvelle de vos inventions se répand dans tous les coins d'Enroth, et les plus sages y prêtent attention. Deyja ne fait bien sûr pas exception. Vous ne me connaissez que comme le souverain d'un pays lointain, mais j'aimerais que vous me considériez comme un collègue, Frederick. Je suis moi-même une sorte d'inventeur. »
```

===== Heroes =====

===== Objects =====

(16, 172, 0) ***Seer's Hut***
Message:

> Il y a environ un siècle, la Guilde des Nécromanciens a créé des amulettes pour amplifier les pouvoirs de ses membres, mais celles-ci ont depuis été perdues. Certaines seraient tombées entre les mains des chefs ogres et gobelins locaux. Les barbares n'avaient probablement aucune idée de ce qu'elles étaient, mais ils ont certainement ressenti la puissance pure des amulettes. Ces artefacts sont susceptibles d'être trouvés dans d'anciens cimetières. M'aideriez-vous à les trouver ? Je peux t'apprendre beaucoup de choses ; autrefois, j'étais aussi doué à l'épée qu'en magie. 

Reviewer: ```suggestion
Il y a environ un siècle, la Guilde des Nécromanciens a créé des amulettes pour amplifier les pouvoirs de ses membres, mais celles-ci ont depuis été perdues. Certaines seraient tombées entre les mains des chefs ogres et gobelins locaux. Les barbares n'avaient probablement aucune idée de ce qu'elles étaient, mais ils ont certainement ressenti la puissance pure des amulettes. Ces artefacts sont susceptibles d'être trouvés dans d'anciens cimetières. M'aideriez-vous à les trouver ? Je peux vous apprendre beaucoup de choses ; autrefois, j'étais aussi doué à l'épée qu'en magie. 
```

> Tu ne les as toujours pas trouvées ? Je suis sûr qu'il y a quelque chose à découvrir dans les anciennes tombes.

Reviewer: ```suggestion
Vous ne les avez toujours pas trouvées ? Je suis sûr qu'il y a quelque chose à découvrir dans les anciennes tombes.
```

Excellent, tout simplement excellent ! Veux-tu commencer tes études maintenant ?

> Il y a environ un siècle, la Guilde des Nécromanciens a créé des amulettes pour amplifier les pouvoirs de ses membres, mais celles-ci ont depuis été perdues. Certaines seraient tombées entre les mains des chefs ogres et gobelins locaux. Les barbares n'avaient probablement aucune idée de ce qu'elles étaient, mais ils ont certainement ressenti la puissance pure des amulettes. Ces artefacts sont susceptibles d'être trouvés dans d'anciens cimetières. M'aideriez-vous à les trouver ? Je peux t'apprendre beaucoup de choses ; autrefois, j'étais aussi doué à l'épée qu'en magie. 

Reviewer: ```suggestion
Il y a environ un siècle, la Guilde des Nécromanciens a créé des amulettes pour amplifier les pouvoirs de ses membres, mais celles-ci ont depuis été perdues. Certaines seraient tombées entre les mains des chefs ogres et gobelins locaux. Les barbares n'avaient probablement aucune idée de ce qu'elles étaient, mais ils ont certainement ressenti la puissance pure des amulettes. Ces artefacts sont susceptibles d'être trouvés dans d'anciens cimetières. M'aideriez-vous à les trouver ? Je peux vous apprendre beaucoup de choses ; autrefois, j'étais aussi doué à l'épée qu'en magie. 
```

Tu ne les as toujours pas trouvées ? Je suis sûr qu'il y a quelque chose à découvrir dans les anciennes tombes.
Excellent, tout simplement excellent ! Veux-tu commencer tes études maintenant ?
Il y a environ un siècle, la Guilde des Nécromanciens a créé des amulettes pour amplifier les pouvoirs de ses membres, mais celles-ci ont depuis été perdues. Certaines seraient tombées entre les mains des chefs ogres et gobelins locaux. Les barbares n'avaient probablement aucune idée de ce qu'elles étaient, mais ils ont certainement ressenti la puissance pure des amulettes. Ces artefacts sont susceptibles d'être trouvés dans d'anciens cimetières. M'aideriez-vous à les trouver ? Je peux t'apprendre beaucoup de choses ; autrefois, j'étais aussi doué à l'épée qu'en magie. 

> Tu ne les as toujours pas trouvées ? Je suis sûr qu'il y a quelque chose à découvrir dans les anciennes tombes.

Reviewer: ```suggestion
Vous ne les avez toujours pas trouvées ? Je suis sûr qu'il y a quelque chose à découvrir dans les anciennes tombes.
```

Excellent, tout simplement excellent ! Veux-tu commencer tes études maintenant ?
Il y a environ un siècle, la Guilde des Nécromanciens a créé des amulettes pour amplifier les pouvoirs de ses membres, mais celles-ci ont depuis été perdues. Certaines seraient tombées entre les mains des chefs ogres et gobelins locaux. Les barbares n'avaient probablement aucune idée de ce qu'elles étaient, mais ils ont certainement ressenti la puissance pure des amulettes. Ces artefacts sont susceptibles d'être trouvés dans d'anciens cimetières. M'aideriez-vous à les trouver ? Je peux t'apprendre beaucoup de choses ; autrefois, j'étais aussi doué à l'épée qu'en magie. 
Tu ne les as toujours pas trouvées ? Je suis sûr qu'il y a quelque chose à découvrir dans les anciennes tombes.

> Excellent, tout simplement excellent ! Veux-tu commencer tes études maintenant ?

Reviewer: ```suggestion
Excellent, tout simplement excellent ! Voulez-vous commencer vos études maintenant ?
```

(11, 99, 0) ***Seer's Hut***
Message:

> Vous entendez des hommes crier. En vous précipitant vers le bruit, vous trouvez un groupe de pistoleros encerclés par des morts-vivants. Vous voulez les aider, mais vous êtes arrêté par un nécromancien qui dirige la charge des soldats morts-vivants. Il dit : « Nous pouvons conclure un marché ! Apportez-moi au moins 200 squelettes et 1 dragon squelette, et je laisserai ces gens partir. »	 Vous hésitez à faire confiance à un nécromancien, mais le combattre mettrait votre armée en danger.

Reviewer: ```suggestion
Vous entendez des hommes crier. En vous précipitant vers le bruit, vous trouvez un groupe de pistoleros encerclés par des morts-vivants. Vous voulez les aider, mais vous êtes arrêté par un nécromancien qui dirige la charge des soldats morts-vivants. Il dit :	 « Nous pouvons conclure un marché ! Apportez-moi au moins 200 squelettes et 1 Dragon Squelette, et je laisserai ces gens partir. »	 Vous hésitez à faire confiance à un nécromancien, mais le combattre mettrait votre armée en danger.
```

> Le nécromancien n'attendra pas éternellement. Apportez-lui 200 squelettes et 1 dragon squelette afin qu'il libère les hommes qu'il a capturés.

Reviewer: ```suggestion
Le nécromancien n'attendra pas éternellement. Apportez-lui 200 squelettes et 1 Dragon Squelette afin qu'il libère les hommes qu'il a capturés.
```

(154, 7, 0) ***Seer's Hut***
Message:

> Un elfe nous accueillit à l'entrée de la cabane : sans aucun doute l'un des elfes noirs de Jadame, la communauté qui dirigeait la guilde marchande d'Alvar. Leurs émissaires venaient souvent à Burton pour faire du commerce, mais j'étais généralement trop absorbé par mes expériences pour aller à leur rencontre. « Enchanté de faire votre connaissance », me dit-il. « Je dirige un comptoir commercial non loin d'ici. Il y a quelque temps, nous avons envoyé une grande caravane à Alvar, mais elle n'est jamais arrivée à destination. Il s'avère qu'elle a été attaquée par une meute de dragons. En général, les dragons jadaméens sont plutôt calmes et s'en prennent rarement à quelqu'un sans raison valable, mais ces créatures... nous ne les avons jamais vues auparavant. Leurs écailles sont d'une teinte orange sale, presque rouille, et leur souffle est sulfureux... Selon l'un de nos guides, ils viennent des îles du sud. Nous ne pouvons pas envoyer de nouvelles caravanes tant que ces dragons sont dans les parages. Aidez-nous à résoudre ce problème et vous serez récompensé. 

Reviewer: leur souffle est sulfureux -> leur souffle est corrosif

Espacement

```suggestion
Un elfe nous accueillit à l'entrée de la cabane : sans aucun doute l'un des elfes noirs de Jadame, la communauté qui dirigeait la guilde marchande d'Alvar. Leurs émissaires venaient souvent à Burton pour faire du commerce, mais j'étais généralement trop absorbé par mes expériences pour aller à leur rencontre.		« Enchanté de faire votre connaissance », me dit-il. « Je dirige un Comptoir Commercial non loin d'ici. Il y a quelque temps, nous avons envoyé une grande caravane à Alvar, mais elle n'est jamais arrivée à destination. Il s'avère qu'elle a été attaquée par une meute de dragons. En général, les dragons jadaméens sont plutôt calmes et s'en prennent rarement à quelqu'un sans raison valable, mais ces créatures... nous ne les avons jamais vues auparavant. Leurs écailles sont d'une teinte orange sale, presque rouille, et leur souffle est corrosif... Selon l'un de nos guides, ils viennent des îles du sud. Nous ne pouvons pas envoyer de nouvelles caravanes tant que ces dragons sont dans les parages. Aidez-nous à résoudre ce problème et vous serez récompensé. 
```

Vous n'avez pas trouvé les dragons ? J'ai entendu dire qu'ils avaient été aperçus pour la dernière fois au sud-est d'ici.
Bravo ! Souhaitez-vous recevoir votre récompense maintenant ?

(32, 66, 0) ***Seer's Hut***
Message:

> Un vieil ogre édenté vous accueille dans une hutte délabrée. « Oh, mes jours de gloire sont loin derrière moi... J'aurais dû le voir venir ; qui voudrait d'un chef incapable ? Ça fait encore mal. Il y avait un guerrier puissant dans ma tribu, Akratos, et j'ai négligé les signes qui indiquaient qu'il complotait pour prendre ma place. Ce serpent ! Il n'a même pas eu la décence de me défier en duel ; non, il a triché. Quand je suis allé chasser dans les montagnes, ses acolytes m'ont tendu une embuscade. Vous voyez, mon bras armé est encore très fort, et j'ai même battu Krellion de Krewlod lui-même dans un combat à mains nues, remarquez... J'ai donné une bonne raclée à ces canailles, mais ils m'ont aussi bien tabassé, et pendant que je me remettais, Akratos a emménagé dans ma maison et a persuadé tout le monde qu'il serait un meilleur chef pour eux que le vieux Deevaz... Maintenant, je ne suis plus le bienvenu dans la tribu. Si j'y vais seul, ils me jetteront une pierre du mur sur la tête, et ce sera fini.		Écoute, humain ! J'ai entendu dire que tu avais battu ce sorcier, Palaemon, donc tu n'es certainement pas une mauviette. Accepterais-tu de chasser cet arriviste d'Akratos de mon village ? Ce vieux ogre n'a pas d'or pour te rembourser, mais j'ai quelque chose de mieux. Tu veux bien m'aider ?

Reviewer: Espacement

Écoutez, humain !

Accepteriez-vous de chasser 

Voulez-vous bien m'aider ?

```suggestion
Un vieil ogre édenté vous accueille dans une hutte délabrée.		« Oh, mes jours de gloire sont loin derrière moi... J'aurais dû le voir venir ; qui voudrait d'un chef incapable ? Ça fait encore mal. Il y avait un guerrier puissant dans ma tribu, Akratos, et j'ai négligé les signes qui indiquaient qu'il complotait pour prendre ma place. Ce serpent ! Il n'a même pas eu la décence de me défier en duel ; non, il a triché. Quand je suis allé chasser dans les montagnes, ses acolytes m'ont tendu une embuscade. Vous voyez, mon bras armé est encore très fort, et j'ai même battu Krellion de Krewlod lui-même dans un combat à mains nues, remarquez... J'ai donné une bonne raclée à ces canailles, mais ils m'ont aussi bien tabassé, et pendant que je me remettais, Akratos a emménagé dans ma maison et a persuadé tout le monde qu'il serait un meilleur chef pour eux que le vieux Deevaz... Maintenant, je ne suis plus le bienvenu dans la tribu. Si j'y vais seul, ils me jetteront une pierre du mur sur la tête, et ce sera fini.		Écoutez, humain ! J'ai entendu dire que tu avais battu ce sorcier, Palaemon, donc tu n'es certainement pas une mauviette. Accepteriez-vous de chasser cet arriviste d'Akratos de mon village ? Ce vieux ogre n'a pas d'or pour te rembourser, mais j'ai quelque chose de mieux. Voulez-vous bien m'aider ?
```

J'ai entendu dire qu'Akratos dirigeait toujours mon village ! Tu ferais mieux de ne pas revenir vers moi avec de mauvaises nouvelles...

> Quelle nouvelle glorieuse ! Voici le trésor dont je t'ai parlé. Des rumeurs circulaient dans ma tribu selon lesquelles il aurait été possédé par un démon, mais je suppose que tu n'es pas du genre timide.	

Reviewer: ```suggestion
Quelle nouvelle glorieuse ! Voici le trésor dont je vous ai parlé. Des rumeurs circulaient dans ma tribu selon lesquelles il aurait été possédé par un démon, mais je suppose que vous n'êtes pas du genre timide.	
```

(12, 50, 0) ***Seer's Hut***
Message:

> Dans la cabane, je me suis retrouvé face à face avec une créature géante à la peau bleu-vert. Un troll, mais certainement pas du genre de ceux que j'avais rencontrés à Eraphia. Les trolls de Jadame n'étaient pas des sauvages sans cervelle comme ceux-là : ils vivaient en communauté, construisaient des maisons en pierre dans leurs villages et faisaient du commerce avec leurs voisins elfes noirs. À Burton et dans les Terres désolées, cependant, les trolls étaient rares ; ils vivaient dans la nature sauvage, loin à l'est. Ce n'était qu'occasionnellement que les marchands elfes les engageaient pour garder leurs caravanes. Il était donc fort probable que le troll que j'avais rencontré ici était un voyageur. Le troll était assis à une immense table dans un coin, plongé dans ses pensées. « Un artisan, hein ? » demanda le troll en haussant les sourcils. « Eh bien, entrez, artisan. » Je suis entré, faisant signe à mes hommes de rester dehors. « Je suis coincé ici depuis des jours ! Ces vers, ou je ne sais quoi, ont détruit la route vers le sud. Je ne sais pas qui a construit cette cabane, peut-être des ogres ou des gobelins. Mais ça me convient très bien, ça me protège du soleil brûlant. — Vous êtes assez loin du désert d'Ironsand, dis-je.	« Je n'aurais jamais quitté mon village et ma tribu, mais il y a quelques mois, mon frère a décidé de s'aventurer sur ces terres. C'était un garçon intelligent, qui avait beaucoup voyagé avec les caravanes elfiques, où il avait appris à lire et commencé à collectionner des histoires et des récits des temps anciens. Quelqu'un lui a raconté une légende au sujet d'une ancienne grotte dans l'ouest de Jadame, qui serait le lieu ancestral de notre peuple. Eh bien, je n'ai plus jamais eu de nouvelles de mon frère. Je suis parti à sa recherche et, en arrivant ici, j'ai rencontré un gobelin qui m'a dit avoir vu un troll mort sur la route, entouré d'une meute de dragons de soufre. Les misérables ! Ils ont fait leur nid non loin d'ici, au sud-est. Je ne suis pas un lâche, mais seul, je ne suis qu'un simple casse-croûte pour ces bêtes. Aidez-moi à venger mon frère, et je vous récompenserai comme je le peux. 

Reviewer: Espacement

j'avais rencontrés à Eraphia -> j'avais rencontrés à Erathia

les Terres désolées -> les Terres Désolées

Un artisan, hein ? -> Un Artificier, hein ?

Eh bien, entrez, artisan -> Eh bien, entrez, Artificier

dragons de soufre -> Dragons Corrosif
```suggestion
Dans la cabane, je me suis retrouvé face à face avec une créature géante à la peau bleu-vert. Un troll, mais certainement pas du genre de ceux que j'avais rencontrés à Erathia. Les trolls de Jadame n'étaient pas des sauvages sans cervelle comme ceux-là : ils vivaient en communauté, construisaient des maisons en pierre dans leurs villages et faisaient du commerce avec leurs voisins elfes noirs. À Burton et dans les Terres Désolées, cependant, les trolls étaient rares ; ils vivaient dans la nature sauvage, loin à l'est. Ce n'était qu'occasionnellement que les marchands elfes les engageaient pour garder leurs caravanes. Il était donc fort probable que le troll que j'avais rencontré ici était un voyageur.		Le troll était assis à une immense table dans un coin, plongé dans ses pensées.		« Un Artificier, hein ? » demanda le troll en haussant les sourcils. « Eh bien, entrez, Artificier. » 		Je suis entré, faisant signe à mes hommes de rester dehors. « Je suis coincé ici depuis des jours ! Ces vers, ou je ne sais quoi, ont détruit la route vers le sud. Je ne sais pas qui a construit cette cabane, peut-être des ogres ou des gobelins. Mais ça me convient très bien, ça me protège du soleil brûlant. 		Vous êtes assez loin du désert d'Ironsand, dis-je. 		« Je n'aurais jamais quitté mon village et ma tribu, mais il y a quelques mois, mon frère a décidé de s'aventurer sur ces terres. C'était un garçon intelligent, qui avait beaucoup voyagé avec les caravanes elfiques, où il avait appris à lire et commencé à collectionner des histoires et des récits des temps anciens. Quelqu'un lui a raconté une légende au sujet d'une ancienne grotte dans l'ouest de Jadame, qui serait le lieu ancestral de notre peuple. Eh bien, je n'ai plus jamais eu de nouvelles de mon frère. Je suis parti à sa recherche et, en arrivant ici, j'ai rencontré un gobelin qui m'a dit avoir vu un troll mort sur la route, entouré d'une meute de Dragons Corrosif. Les misérables ! Ils ont fait leur nid non loin d'ici, au sud-est. Je ne suis pas un lâche, mais seul, je ne suis qu'un simple casse-croûte pour ces bêtes. Aidez-moi à venger mon frère, et je vous récompenserai comme je le peux. 
```
```suggestion
Dans la cabane, je me suis retrouvé face à face avec une créature géante à la peau bleu-vert. Un troll, mais certainement pas du genre de ceux que j'avais rencontrés à Eraphia. Les trolls de Jadame n'étaient pas des sauvages sans cervelle comme ceux-là : ils vivaient en communauté, construisaient des maisons en pierre dans leurs villages et faisaient du commerce avec leurs voisins elfes noirs. À Burton et dans les Terres désolées, cependant, les trolls étaient rares ; ils vivaient dans la nature sauvage, loin à l'est. Ce n'était qu'occasionnellement que les marchands elfes les engageaient pour garder leurs caravanes. Il était donc fort probable que le troll que j'avais rencontré ici était un voyageur. Le troll était assis à une immense table dans un coin, plongé dans ses pensées. « Un artisan, hein ? » demanda le troll en haussant les sourcils. « Eh bien, entrez, artisan. » Je suis entré, faisant signe à mes hommes de rester dehors. « Je suis coincé ici depuis des jours ! Ces vers, ou je ne sais quoi, ont détruit la route vers le sud. Je ne sais pas qui a construit cette cabane, peut-être des ogres ou des gobelins. Mais ça me convient très bien, ça me protège du soleil brûlant. — Vous êtes assez loin du désert d'Ironsand, dis-je.	« Je n'aurais jamais quitté mon village et ma tribu, mais il y a quelques mois, mon frère a décidé de s'aventurer sur ces terres. C'était un garçon intelligent, qui avait beaucoup voyagé avec les caravanes elfiques, où il avait appris à lire et commencé à collectionner des histoires et des récits des temps anciens. Quelqu'un lui a raconté une légende au sujet d'une ancienne grotte dans l'ouest de Jadame, qui serait le lieu ancestral de notre peuple. Eh bien, je n'ai plus jamais eu de nouvelles de mon frère. Je suis parti à sa recherche et, en arrivant ici, j'ai rencontré un gobelin qui m'a dit avoir vu un troll mort sur la route, entouré d'une meute de dragons de soufre. Les misérables ! Ils ont fait leur nid non loin d'ici, au sud-est. Je ne suis pas un lâche, mais seul, je ne suis qu'un simple casse-croûte pour ces bêtes. Aidez-moi à venger mon frère, et je vous récompenserai comme je le peux. 
```

Je sais que les dragons de soufre sont toujours là. Je ne peux pas retourner dans mon village couvert de honte tant que mon frère n'aura pas été vengé !
Les dragons de soufre sont morts ? Je n'ai jamais douté de vous. Tenez, prenez cette épée. Il y a plusieurs décennies, mon grand-père l'a déterrée au milieu du désert. Quand nous étions petits, il nous racontait des histoires effrayantes, à mon frère et moi, disant qu'elle était imprégnée de magie diabolique, mais je ne croyais pas à ces histoires à l'époque, et je n'y crois toujours pas aujourd'hui. Souhaitez-vous accepter l'épée flamboyante que vous tend le troll ?

(109, 65, 0) ***Sign***
Message:
Comptoir commercial de la guilde marchande d'Alvar

(138, 85, 0) ***Sign***
Message:
Arène de la gloire

(144, 83, 0) ***Seer's Hut***
Message:

> Lorsque vous avez aperçu une vieille cabane creusée à l'intérieur d'un immense arbre mort, vous vous attendiez à rencontrer un troll ou peut-être un ogre, mais c'est un elfe qui vous a accueilli. Cela n'aurait pas été très surprenant s'il s'était agi d'un elfe des bois, mais c'était un elfe noir, qui semblait être un mage fortuné. Que faisait-il dans ce trou ? « Je suis un sorcier de la Guilde élémentaire d'Alvar », s'est présenté l'elfe. « Nous nous intéressons à toutes sortes de créatures élémentaires. Si tout mage suffisamment puissant peut invoquer des élémentaires, les génies et les éfrits sont une tout autre affaire, et nous les voulons pour nos recherches. J'ai entendu dire que les génies sont faciles à trouver dans la lointaine Bracada, mais ils accepteraient difficilement de voyager jusqu'aux confins du monde à la demande d'un elfe inconnu. Récemment, cependant, nous avons entendu une rumeur selon laquelle on pouvait rencontrer des génies et des éfrits dans une ancienne tombe non loin d'ici.		Je me suis rendu dans ces contrées pour étudier les wyvernes locales. J'ai un faible pour ces créatures, même si elles ne correspondent pas exactement à mon profil scientifique. Quoi qu'il en soit, j'étais déjà dans les environs, alors je suis venu ici. Malheureusement, la tombe s'est avérée être un endroit bien trop dangereux, même pour un mage expérimenté comme moi. Je vois cependant à votre apparence que vous avez l'intention de vous y rendre. Peut-être trouverez-vous des génies et des éfrits prêts à m'accompagner à Alvar ? Ah, oui, et si vous trouvez de jeunes wyvernes venimeuses, amenez-les-moi. Je n'ai pas envie de m'aventurer moi-même dans leur nid, et je n'ai trouvé aucun trappeur ici disposé à me les vendre.

Reviewer: Espacement

Capitale

Effrits

```suggestion
Lorsque vous avez aperçu une vieille cabane creusée à l'intérieur d'un immense arbre mort, vous vous attendiez à rencontrer un troll ou peut-être un ogre, mais c'est un elfe qui vous a accueilli. Cela n'aurait pas été très surprenant s'il s'était agi d'un elfe des bois, mais c'était un elfe noir, qui semblait être un mage fortuné. Que faisait-il dans ce trou ?		« Je suis un sorcier de la Guilde élémentaire d'Alvar », s'est présenté l'elfe. « Nous nous intéressons à toutes sortes de créatures élémentaires. Si tout mage suffisamment puissant peut invoquer des élémentaires, les Génies et les Effrits sont une tout autre affaire, et nous les voulons pour nos recherches. J'ai entendu dire que les Génies sont faciles à trouver dans la lointaine Bracada, mais ils accepteraient difficilement de voyager jusqu'aux confins du monde à la demande d'un elfe inconnu. Récemment, cependant, nous avons entendu une rumeur selon laquelle on pouvait rencontrer des génies et des éfrits dans une ancienne tombe non loin d'ici.		Je me suis rendu dans ces contrées pour étudier les Wyvernes locales. J'ai un faible pour ces créatures, même si elles ne correspondent pas exactement à mon profil scientifique. Quoi qu'il en soit, j'étais déjà dans les environs, alors je suis venu ici. Malheureusement, la tombe s'est avérée être un endroit bien trop dangereux, même pour un Mage expérimenté comme moi. Je vois cependant à votre apparence que vous avez l'intention de vous y rendre. Peut-être trouverez-vous des Génies et des Effrits prêts à m'accompagner à Alvar ? Ah, oui, et si vous trouvez de jeunes Wyvernes venimeuses, amenez-les-moi. Je n'ai pas envie de m'aventurer moi-même dans leur nid, et je n'ai trouvé aucun trappeur ici disposé à me les vendre.
```

Les chevaliers sont inflexibles. Ils veulent exactement 25 cristaux, 25 gemmes et 50 000 pièces d'or, pas une pièce de moins !
Grommelant d'un air boudeur, l'un des chevaliers va libérer les prisonniers. Voulez-vous donner 25 cristaux, 25 gemmes et 50 000 pièces d'or aux chevaliers ?

(31, 159, 0) ***Bounty Hunter***
Message:
Parmi les prisonniers, vous reconnaissez un pistolero familier de Burton. « Frederick ! » s'exclame-t-il. « Je ne pensais pas te revoir un jour. Nous avons été piégés après l'éboulement, puis ces tueurs de dragons sont apparus de nulle part et nous ont accusés d'avoir causé la catastrophe. D'après ce que m'ont dit les soldats, leur commandant, Volothomon, chassait des dragons noirs dans les montagnes au nord d'une certaine tombe, et il semble être tombé dans le même piège que notre groupe. »

> 

Reviewer: a retiré l'espace

(110, 156, 0) ***Quest Guard***
Message:
À l'extérieur de la mine, vous rencontrez un homme qui travaille sur des pièces d'automates ; cependant, l'artisan semble plutôt inexpérimenté, et l'assemblage de nouvelles machines prend du temps, ce que vous n'avez pas. L'artisan vous dit que si vous lui apportez 25 mercure, 50 minerai et 25 soufre, il pourra construire des automates plus rapidement.

> Malheureusement, sans ressources, vous ne pouvez rien faire pour aider l'artisan. L'usine de la ville est détruite et il n'y a aucun moyen d'assembler les automates plus rapidement.

Reviewer: ```suggestion
Malheureusement, sans ressources, vous ne pouvez rien faire pour aider l'Artificier. La Fabrique de la ville est détruite et il n'y a aucun moyen d'assembler les automates plus rapidement.
```

Avec votre aide, l'inventeur peut rapidement assembler 16 automates. Souhaitez-vous partager certaines de vos ressources avec lui ?

(86, 89, 0) ***Knight***
Name:
Volothomon

(11, 152, 0) ***Harpy Hag***
Message:
Lorsque le dernier vampire s'est effondré, nous avons été surpris de trouver plusieurs dizaines de harpies languissant dans des cages. Nous n'avons aucune idée de ce que les vampires leur réservaient !

(111, 123, 0) ***Seer's Hut***
Message:

> Récemment, notre village situé au nord d'ici a été détruit par une éruption volcanique. Nous pensions qu'il s'agissait simplement d'une catastrophe naturelle de plus, car elles sont fréquentes dans cette région du monde. Cependant, le mage à qui j'ai parlé affirme que c'est l'œuvre du maléfique esprit de la terre, Ulkumi, qui habite quelque part dans ces montagnes. Je ne sais pas si c'est vrai ou non, mais si le mage ne mentait pas, l'esprit est toujours dans les parages. Qui sait quels malheurs il va nous causer ?     Je t'en prie, trouve ce monstre et délivre-nous de cette menace !

Reviewer: ```suggestion
Récemment, notre village situé au nord d'ici a été détruit par une éruption volcanique. Nous pensions qu'il s'agissait simplement d'une catastrophe naturelle de plus, car elles sont fréquentes dans cette région du monde. Cependant, le mage à qui j'ai parlé affirme que c'est l'œuvre du maléfique esprit de la terre, Ulkumi, qui habite quelque part dans ces montagnes. Je ne sais pas si c'est vrai ou non, mais si le mage ne mentait pas, l'esprit est toujours dans les parages. Qui sait quels malheurs il va nous causer ?     Je vous en prie, trouvez ce monstre et délivrez-nous de cette menace !
```

Tu n'as toujours pas trouvé l'esprit de la terre ? Peut-être qu'il n'existe pas...

> Tu dis qu'il y avait une horde d'élémentaires ? Je ne pensais pas qu'ils étaient si nombreux. Quoi qu'il en soit, Ulkumi est vaincu. Prends cet anneau de vie. Il n'a pas sauvé notre village, mais il pourra peut-être t'aider.

Reviewer: ```suggestion
Vous dîtes qu'il y avait une horde d'élémentaires ? Je ne pensais pas qu'ils étaient si nombreux. Quoi qu'il en soit, Ulkumi est vaincu. Prends cet Anneau de Vie. Il n'a pas sauvé notre village, mais il pourra peut-être t'aider.
```

(174, 17, 0) ***Seer's Hut***
Message:

> Un elfe noir, qui semblait être un riche marchand, vous a rencontré près de la cabane. Il a ordonné à son garde du corps minotaure de s'écarter. « On m'a toujours dit de ne jamais faire confiance aux ogres... Pourquoi n'écoute-je jamais les bons conseils ? Je fais le commerce de cristal de roche et de pierres précieuses depuis des années, et je me rends souvent dans ces montagnes. Le paysage ici est pittoresque, et c'est un plaisir de transporter des marchandises dans cette région, mais il y a un problème : les ogres. Ils ont immédiatement commencé à attaquer mes caravanes. J'ai eu de la chance au début : j'ai conclu un accord avec l'un de leurs chefs, et pendant un certain temps, les pillages ont cessé. Cependant, il y a quelques mois, l'une de mes caravanes, chargée à ras bord de marchandises précieuses, a été saccagée par ces maudits peaux-noires. On m'a dit que les ogres avaient désormais un nouveau chef, qui se moque éperdument de l'accord que j'avais conclu avec le vieux Deevaz. On dit que ce nouveau venu s'appelle Akratos. Son repaire se trouve haut dans les montagnes, à l'ouest d'ici, mais je sais avec certitude qu'on peut y accéder par un monolithe. Pouvez-vous m'aider à régler ce problème ? Je déteste demander de l'aide à des étrangers, mais je sais que vous avez éliminé Dryope. Quelques barbares ne devraient pas vous poser de problème, même s'ils sont nombreux. En échange de ce service, je vous donnerai un artefact qui, je pense, vous sera très utile.

Reviewer: ```suggestion
Un elfe noir, qui semblait être un riche marchand, vous a rencontré près de la cabane. Il a ordonné à son garde du corps minotaure de s'écarter.		« On m'a toujours dit de ne jamais faire confiance aux ogres... Pourquoi n'écoute-je jamais les bons conseils ? Je fais le commerce de cristal de roche et de pierres précieuses depuis des années, et je me rends souvent dans ces montagnes. Le paysage ici est pittoresque, et c'est un plaisir de transporter des marchandises dans cette région, mais il y a un problème : les ogres. Ils ont immédiatement commencé à attaquer mes caravanes. J'ai eu de la chance au début : j'ai conclu un accord avec l'un de leurs chefs, et pendant un certain temps, les pillages ont cessé. Cependant, il y a quelques mois, l'une de mes caravanes, chargée à ras bord de marchandises précieuses, a été saccagée par ces maudits peaux-noires. On m'a dit que les ogres avaient désormais un nouveau chef, qui se moque éperdument de l'accord que j'avais conclu avec le vieux Deevaz. On dit que ce nouveau venu s'appelle Akratos. Son repaire se trouve haut dans les montagnes, à l'ouest d'ici, mais je sais avec certitude qu'on peut y accéder par un monolithe.		Pouvez-vous m'aider à régler ce problème ? Je déteste demander de l'aide à des étrangers, mais je sais que vous avez éliminé Dryope. Quelques barbares ne devraient pas vous poser de problème, même s'ils sont nombreux. En échange de ce service, je vous donnerai un artefact qui, je pense, vous sera très utile.
```

(92, 150, 0) ***Quest Guard***
Message:
Ce portail a été construit récemment par des artisans. Il menait à une grotte où ils expérimentaient de nouveaux types d'armes, mais quelque chose a mal tourné : il s'est brisé. Le portail scintille et clignote de manière menaçante, il vaut mieux éviter d'essayer d'y entrer. Si je pouvais trouver des artefacts capables de supprimer la magie, je serais en mesure de désactiver le portail et de le restaurer.
Ce portail a été construit récemment par des artisans. Il menait à une grotte où ils expérimentaient de nouveaux types d'armes, mais quelque chose a mal tourné : il s'est brisé. Le portail scintille et clignote de manière menaçante : mieux vaut éviter d'essayer d'y entrer. Si je pouvais trouver des artefacts capables de supprimer la magie, je pourrais désactiver le portail et le restaurer.
Restaurer le portail ?

(163, 7, 0) ***Gunslinger***
Message:

> Alors que nous traversions la garnison, nous avons remarqué d'énormes cages remplies de personnes : des mercenaires et des mécaniciens. J'ai fait signe aux soldats de libérer les prisonniers. Après avoir discuté avec l'un d'entre eux, il s'est avéré qu'ils avaient été capturés par une bande errante de chevaliers noirs. Selon le capitaine mercenaire, les chevaliers cherchaient une ancienne crypte, mais il n'a pas pu fournir d'autres informations utiles.

Reviewer: ```suggestion
Alors que nous traversions la garnison, nous avons remarqué d'énormes cages remplies de personnes : des mercenaires et des mécaniciens. J'ai fait signe aux soldats de libérer les prisonniers. Après avoir discuté avec l'un d'entre eux, il s'est avéré qu'ils avaient été capturés par une bande errante de Chevaliers Noirs. Selon le capitaine mercenaire, les chevaliers cherchaient une ancienne crypte, mais il n'a pas pu fournir d'autres informations utiles.
```

(105, 160, 0) ***Quest Guard***
Message:

> Les mécaniciens qui travaillent dans le village situé à l'extérieur de Volta refusent de se joindre à l'expédition. Leur usine a été détruite après l'attaque des vers des sables, et ils ne bougeront pas tant qu'elle n'aura pas été reconstruite. Après leur avoir parlé, vous découvrez qu'il faudra 30 de bois, 30 de pierre, 20 de soufre et 20 cristaux pour reconstruire toutes les forges.

Reviewer: ```suggestion
Les mécaniciens qui travaillent dans le village situé à l'extérieur de Volta refusent de se joindre à l'expédition. Leur Manufacture a été détruite après l'attaque des Vers des Sables, et ils ne bougeront pas tant qu'elle n'aura pas été reconstruite. Après leur avoir parlé, vous découvrez qu'il faudra 30 de bois, 30 de pierre, 20 de soufre et 20 cristaux pour reconstruire toutes les forges.
```

(84, 116, 0) ***Quest Guard***
Message:
Le chemin menant à cette partie du tombeau a été scellé par magie par les maîtres génies. Tant qu'ils sont là, il est impossible d'ouvrir cette porte.

> Le chemin menant à cette partie du tombeau a été scellé par magie par les maîtres génies. Tant qu'ils sont là, il est impossible d'ouvrir cette porte.

Reviewer: ```suggestion
Le chemin menant à cette partie du tombeau a été scellé par magie par les Maîtres Génies. Tant qu'ils sont là, il est impossible d'ouvrir cette porte.
```

Tant que nous n'aurons pas vaincu Fulghea et son armée dans la grotte, l'elfe noir ne voudra pas traiter avec nous.
Vous dites que c'est sûr là-dedans maintenant ? C'est une bonne nouvelle. La grotte est à nouveau à nous ! L'elfe noir se frotte les mains. Et voici la bague dont je vous ai parlé. Vous la voulez ?

(73, 177, 0) ***Quest Gate***
Message:
Vous apercevez le chantier naval devant vous.

> Vous apercevez le chantier naval devant vous.

Reviewer: ```suggestion
Vous apercevez le Chantier Aéronaval devant vous.
```

Souhaitez-vous continuer ?

(76, 176, 0) ***Halfling***
Message:

> Le chantier naval des dirigeables se profilait devant nous, et à l'entrée se trouvait un hobbit portant un casque muni de grandes lunettes. Il nous fit un signe amical de la main et nous invita à entrer. Notre opération de collecte de ressources avait déjà pris trop de temps, et cette recherche allait nous retarder encore davantage... Alors que nous marchions vers les dirigeables, Kastore ne dit pas un mot. Il semblait plongé dans ses pensées pendant le vol d'essai. Lorsque la nacelle toucha à nouveau le sol, Kastore se tourna vers moi : « J'aimerais faire partie de votre expédition. Vous avez un grand potentiel, et je m'en voudrais de ne pas poursuivre notre relation. Vous avez déjà remarqué que je ne suis pas étranger aux aventures moi-même. Les souvenirs de mes exploits passés me touchent profondément. Vous n'allez pas refuser ma demande, n'est-ce pas ? » « Bien sûr... Nous partons dans une heure », marmonnai-je, perplexe. Je ne m'attendais pas à ce que Kastore se joigne à l'expédition. Cependant, mon esprit fourmillait de questions auxquelles j'espérais que l'elfe apporterait des réponses. Bien sûr, un autre sorcier puissant à nos côtés serait également le bienvenu.

Reviewer: ```suggestion
Le Chantier Aéronaval se profilait devant nous, et à l'entrée se trouvait un hobbit portant un casque muni de grandes lunettes. Il nous fit un signe amical de la main et nous invita à entrer.		Notre opération de collecte de ressources avait déjà pris trop de temps, et cette recherche allait nous retarder encore davantage...		Alors que nous marchions vers les dirigeables, Kastore ne dit pas un mot. Il semblait plongé dans ses pensées pendant le vol d'essai. Lorsque la nacelle toucha à nouveau le sol, Kastore se tourna vers moi :		« J'aimerais faire partie de votre expédition. Vous avez un grand potentiel, et je m'en voudrais de ne pas poursuivre notre relation. Vous avez déjà remarqué que je ne suis pas étranger aux aventures moi-même. Les souvenirs de mes exploits passés me touchent profondément. Vous n'allez pas refuser ma demande, n'est-ce pas ? »		« Bien sûr... Nous partons dans une heure », marmonnai-je, perplexe.		Je ne m'attendais pas à ce que Kastore se joigne à l'expédition. Cependant, mon esprit fourmillait de questions auxquelles j'espérais que l'elfe apporterait des réponses. Bien sûr, un autre sorcier puissant à nos côtés serait également le bienvenu.
```

L'armée des morts-vivants garde toujours le portail. Tant que les dragons de la Salle des Illusions ne sont pas détruits, il vaut mieux ne pas prendre le risque de s'en mêler.
« Les dragons de la Salle des Illusions... ont disparu ? demanda le liche, la voix teintée de doute. « Bon, » si un crâne proprement décortiqué pouvait sourire, vous jureriez qu'il venait de le faire, « je n'ai aucune raison de me méfier de vous. Vous pouvez vous rendre au portail, et nous allons prendre congé. Nous n'avons reçu aucun nouvel ordre de Trophon, et nous n'apprécions pas d'être laissés ici sans rien faire. »        Souhaitez-vous passer ?

(8, 152, 0) ***Quest Guard***
Message:

> Ces portes sont gardées par des liches puissantes. Elles sont réticentes à vous parler, mais finissent par vous dire qu'elles seraient disposées à ouvrir le passage à celui qui leur rendra leur Emblème du Faussaire. L'une des liches vous dit que l'emblème a été perdu après une attaque d'un dragon d'or qui vit dans une tour voisine.

Reviewer: ```suggestion
Ces portes sont gardées par des liches puissantes. Elles sont réticentes à vous parler, mais finissent par vous dire qu'elles seraient disposées à ouvrir le passage à celui qui leur rendra leur Emblème du Faussaire. L'une des liches vous dit que l'emblème a été perdu après une attaque d'un Dragon d'Or qui vit dans une tour voisine.
```

> Ces portes sont gardées par des liches puissantes. Elles sont réticentes à vous parler, mais finissent par vous dire qu'elles sont prêtes à ouvrir le passage à celui qui leur rendra leur Emblème du Faussaire. L'une des liches vous dit que l'emblème a été perdu après une attaque d'un dragon d'or qui vit dans une tour voisine.

Reviewer: ```suggestion
Ces portes sont gardées par des liches puissantes. Elles sont réticentes à vous parler, mais finissent par vous dire qu'elles sont prêtes à ouvrir le passage à celui qui leur rendra leur Emblème du Faussaire. L'une des liches vous dit que l'emblème a été perdu après une attaque d'un Dragon d'Or qui vit dans une tour voisine.
```

Les liches vous prennent l'emblème et vous laissent passer à contrecœur.     Souhaitez-vous passer ?

(94, 104, 0) ***Event***
Message:
Ce qui se dressait devant nous était l'entrée d'une immense tombe... si tant est qu'il s'agissait bien d'une tombe. Kastore, qui m'avait suivi avec son expression indifférente, voire aigre, habituelle, s'est soudainement animé. Il a regardé avec un vif intérêt les symboles étranges disséminés ici et là pendant un moment. Puis il a dit : « Eh bien, c'est intéressant. J'ai déjà vu des tombes comme celle-ci. » Sur ces mots, Kastore sortit de sa ceinture un objet dont je ne connaissais pas l'utilité et fit quelques pas en avant. Cette chose avait depuis longtemps attiré mon attention. Qu'était-ce donc ? Un instrument scientifique que même moi je ne connaissais pas ? Un héritage familial ? Qui sait...		L'elfe agita son objet d'un côté à l'autre pendant un moment, scrutant les couloirs sombres de la tombe, puis finit par expirer. « Il ne semble y avoir personne à l'intérieur, nous pouvons entrer. Mais cela ne signifie pas pour autant que nous pouvons baisser notre garde », dit Kastore d'un ton didactique.

(38, 157, 0) ***Seer's Hut***
Message:

> Au pied des montagnes, vous trouvez un petit camp minier et un groupe d'ingénieurs armés. Lorsqu'ils apprennent la disparition de l'expédition d'Henrietta, ils se déclarent prêts à vous accompagner, mais ils ne peuvent malheureusement pas laisser le camp sans surveillance, car le vampire Mormolykos a récemment été aperçu dans le village et plusieurs mineurs ont été victimes de ses attaques. Si vous parvenez à retrouver le vampire et à le détruire, les ingénieurs se joindront à vous dans votre quête.

Reviewer: ```suggestion
Au pied des montagnes, vous trouvez un petit camp minier et un groupe d'ingénieurs armés. Lorsqu'ils apprennent la disparition de l'expédition d'Henrietta, ils se déclarent prêts à vous accompagner, mais ils ne peuvent malheureusement pas laisser le camp sans surveillance, car le vampire Mormolykos a récemment été aperçu dans le village et plusieurs mineurs ont été victimes de ses attaques. Si vous parvenez à retrouver le vampire et à le détruire, les Ingénieurs se joindront à vous dans votre quête.
```

Les ingénieurs estiment que le village doit encore être protégé. Le vampire pourrait revenir.

> Lorsque les ingénieurs voient la capuche de Mormolykos, ils se déclarent immédiatement prêts à vous accompagner. Ils sont surpris d'apprendre que le vampire a été trouvé si loin, mais ils acquiescent avec compréhension lorsqu'ils entendent votre histoire au sujet du vieux manoir.    Voulez-vous emmener les ingénieurs avec vous ?

Reviewer: ```suggestion
Lorsque les Ingénieurs voient la capuche de Mormolykos, ils se déclarent immédiatement prêts à vous accompagner. Ils sont surpris d'apprendre que le vampire a été trouvé si loin, mais ils acquiescent avec compréhension lorsqu'ils entendent votre histoire au sujet du vieux manoir.    Voulez-vous emmener les ingénieurs avec vous ?
```

L'entrée du tombeau se trouve au nord d'ici. L'expédition d'Henrietta est passée par là il n'y a pas si longtemps, mais aujourd'hui, ce chemin à travers les montagnes est enseveli sous les rochers.	Peut-être que si vous trouviez un artefact puissant capable d'amplifier la magie de la Terre, vous pourriez briser les rochers et passer.
En utilisant toute la puissance du Globe des Pierres, vous pourrez passer. Malheureusement, si vous le faites, le globe perdra son pouvoir magique. Voulez-vous sacrifier le globe et passer ?

(19, 145, 0) ***Quest Guard***
Message:
Il s'agit d'un portail bidirectionnel, mais vous ne pouvez ouvrir l'entrée que d'un seul côté.
Il s'agit d'un portail bidirectionnel, mais vous ne pouvez ouvrir l'entrée que d'un seul côté.
Voulez-vous ouvrir le portail ?

(175, 100, 0) ***Quest Guard***
Message:
Le laboratoire d'Agar se trouve quelque part au nord d'ici. Cependant, n'oublions pas que notre objectif principal est de retrouver l'expédition d'Henrietta.
Notre tâche consiste désormais à explorer la tombe.

> Nos recherches dans la tombe ont été vaines jusqu'à présent, mais il reste encore beaucoup à explorer. Maintenant que les vers des sables ont détruit les passages menant aux profondeurs, il est impossible d'aller plus loin. La seule solution est de trouver Agar. Il pourra peut-être nous aider.    Voulez-vous aller vers le nord ?

Reviewer: ```suggestion
Nos recherches dans la tombe ont été vaines jusqu'à présent, mais il reste encore beaucoup à explorer. Maintenant que les Vers des Sables ont détruit les passages menant aux profondeurs, il est impossible d'aller plus loin. La seule solution est de trouver Agar. Il pourra peut-être nous aider.    Voulez-vous aller vers le nord ?
```

(90, 141, 0) ***Cleric***
Name:
Tagete

(79, 166, 0) ***Event***
Message:
Sur la route menant aux montagnes, nous avons rencontré un nain solitaire, un mineur à première vue, qui semblait très inquiet. En lui parlant, j'ai appris que le vampire Skoulex venait d'arriver de Shadowspire, accompagné d'une horde de morts-vivants. Selon le mineur, les morts-vivants l'avaient presque rattrapé, mais il avait réussi à s'enfuir et à se cacher dans les falaises. Pendant la nuit, alors que le nain traversait l'obscurité pour contourner le camp des morts-vivants, il a entendu des bribes de conversation entre le liche et le chevalier de la mort. Il semblait qu'ils étaient venus au Ravage Roaming à la recherche d'un autre vampire, une créature très ancienne dont ils parlaient à voix basse. Je me demande de qui ils parlaient...		Quoi qu'il en soit, je n'ai pas le temps d'essayer de comprendre cette histoire. Ces liches sont arrivées au mauvais moment ; nous allons devoir les combattre.

(86, 90, 0) ***Event***
Message:

> Le campement des chasseurs de dragons a attiré notre attention de loin. En m'approchant, j'ai pu voir clairement le camp, les sentinelles postées autour et les griffons planant haut dans le ciel. Un cavalier en armure est sorti du camp et nous a transpercés d'un regard sévère. « Je m'appelle Volothomon », a déclaré le chevalier. « Vous avez peut-être entendu parler de nous ; nous sommes venus à Jadame pour mettre fin une fois pour toutes à la menace des dragons sur ce continent. Mes frères d'armes ont combattu les dragons de Nighon pendant la guerre de Restauration, et nous avons une grande expérience dans ce domaine. Notre principale forteresse est loin de ces terres, mais nous avons entendu dire que des dragons se trouvaient également dans votre région. Alors, qui êtes-vous et que recherchez-vous ? Nous sommes... des mages. Les dragons ne nous intéressent guère. Tout ce dont nous avons besoin, ce sont quelques vieilles clés. Le chevalier fronça les sourcils. Des clés, dites-vous ? Vous voulez parler de celles-ci ?		Il sortit un gros trousseau de clés de sa ceinture et le secoua.		« Nous les avons achetées à la foire des Elfes Noirs, et vous pouvez certainement imaginer combien elles nous ont coûté cher. Si les trésors cachés derrière les serrures qu'elles ouvrent représentent ne serait-ce qu'un centième de ce que ces elfes ont prétendu... J'équiperai mes troupes suffisamment bien pour débarrasser le monde entier de cette vermine ailée sans plus attendre. Et si vous pensez que je suis disposé à vous céder ces clés, alors j'en déduis que vous cherchez à m'insulter ! Mais tu as de la chance : je ne suis pas d'humeur à donner des leçons de tact pour le moment. Quoi qu'il en soit, voudrais-tu nous acheter des écailles ou des os de dragon ? Nous en avons en abondance, et je sais que vous, les mages, aimez ce genre de choses. Je te conseille sincèrement d'oublier les clés, cependant ! »		Je murmurai que j'allais certainement réfléchir à cette offre, et sous le regard hautain du chevalier, je laissai Kastore m'éloigner. « Ça ne sert à rien de marchander avec cet imbécile, Frederick ; tu ne le vois pas toi-même ? Je acquiesçai avec un soupir. Je n'avais rien contre les chevaliers, mais nous n'avions tout simplement pas d'autre choix : nous allions devoir nous emparer des clés par la force.

Reviewer: ```suggestion
Le campement des chasseurs de dragons a attiré notre attention de loin. En m'approchant, j'ai pu voir clairement le camp, les sentinelles postées autour et les Griffons planant haut dans le ciel. Un cavalier en armure est sorti du camp et nous a transpercés d'un regard sévère. « Je m'appelle Volothomon », a déclaré le Chevalier. « Vous avez peut-être entendu parler de nous ; nous sommes venus à Jadame pour mettre fin une fois pour toutes à la menace des dragons sur ce continent. Mes frères d'armes ont combattu les dragons de Nighon pendant la Guerre de Restauration, et nous avons une grande expérience dans ce domaine. Notre principale forteresse est loin de ces terres, mais nous avons entendu dire que des dragons se trouvaient également dans votre région. Alors, qui êtes-vous et que recherchez-vous ? Nous sommes... des mages. Les dragons ne nous intéressent guère. Tout ce dont nous avons besoin, ce sont quelques vieilles clés. »		Le chevalier fronça les sourcils. « Des clés, dites-vous ? Vous voulez parler de celles-ci ? »		Il sortit un gros trousseau de clés de sa ceinture et le secoua.		« Nous les avons achetées à la foire des Elfes Noirs, et vous pouvez certainement imaginer combien elles nous ont coûté cher. Si les trésors cachés derrière les serrures qu'elles ouvrent représentent ne serait-ce qu'un centième de ce que ces elfes ont prétendu... J'équiperai mes troupes suffisamment bien pour débarrasser le monde entier de cette vermine ailée sans plus attendre. Et si vous pensez que je suis disposé à vous céder ces clés, alors j'en déduis que vous cherchez à m'insulter ! Mais vous avez de la chance : je ne suis pas d'humeur à donner des leçons de tact pour le moment. Quoi qu'il en soit, voudriez-vous nous acheter des écailles ou des os de dragon ? Nous en avons en abondance, et je sais que vous, les mages, aimez ce genre de choses. Je vous conseille sincèrement d'oublier les clés, cependant ! »		Je murmurai que j'allais certainement réfléchir à cette offre, et sous le regard hautain du chevalier, je laissai Kastore m'éloigner.		« Ça ne sert à rien de marchander avec cet imbécile, Frederick ; vous ne le voyez pas vous-même ? Je acquiesçai avec un soupir. Je n'avais rien contre les Chevaliers, mais nous n'avions tout simplement pas d'autre choix : nous allions devoir nous emparer des clés par la force.
```

> 

Reviewer: espace à retiré

(51, 64, 1) ***Quest Gate***
Message:
Le passage vers le laboratoire d'Agar est devant vous, mais c'est un aller simple.
Le passage vers le laboratoire d'Agar est devant vous, mais c'est un aller simple.
Voulez-vous passer ?

(62, 22, 1) ***Seer's Hut***
Message:

> Le laboratoire de la nécromancienne Meline était un lieu très imposant. Dans un coin, je pouvais voir d'énormes flacons remplis de liquides bouillonnants ; des récipients étranges pendaient à des chaînes fixées au plafond ; et devant, dans une large caverne, je pouvais voir trois squelettes de dragons enchaînés ensemble. Sur l'une des tables jonchées de traités magiques, j'ai remarqué le journal de la nécromancienne.	Les dernières entrées étaient éloquentes : « ... 5 février. Une autre expérience a échoué. Les bocaux contenant les âmes ont été détruits et les dragons sont morts pour de bon. Peut-être que les dragons noirs ne conviennent pas à cet usage. 24 février. Mes morts-vivants ont réussi à capturer trois dragons verts dans la grotte voisine. Il n'y a aucune raison de se décourager, je suis sûr que je vais réussir cette fois-ci.		25 février. Une nouvelle erreur de calcul, un nouvel échec. Je vais devoir envoyer les morts-vivants à la recherche d'un nouveau dragon. De plus, l'un des fantômes m'a dit que le vieux Trophon fouinait dans les parages. C'est bien la dernière chose dont j'ai besoin. Je ne peux pas me permettre de perdre ce laboratoire aussi... » « Mais que se passe-t-il ici ? » m'écriai-je, surpris, en refermant le journal et en regardant autour de moi.		Agar ne semblait pas perplexe face à cet étrange laboratoire. « Ce qui se passe ici est très clair. Quelqu'un essayait de créer un dragon fantome. Encore une fois. Le fait établi de longue date que cela est totalement irréalisable ne semble empêcher personne d'essayer. Vous savez, c'est le même problème que certains artisans peu sensés et leurs idées saugrenues de machine à mouvement perpétuel. Sauf que leurs inventions, au pire, explosent simplement, alors que cette chose-là peut produire des fruits vraiment pourris ! » « Oh, je vous en prie ! À Bracada, mes collègues m'ont aussi dit que... »		Me souvenant d'un événement passé, je faillis m'offusquer des propos insolents du vieil homme, mais Kastore, qui était resté silencieux jusqu'alors, intervint dans la conversation. « Si nous avions moins de soucis à nous faire en ce moment, je vous parlerais du mouvement perpétuel, mes chers... érudits. Mais regardez, qu'est-ce que c'est que ça ? » Son attention fut attirée par une porte fermée dans un coin du laboratoire.		« Je parie que c'est une salle au trésor. Et scellée par magie, en plus ! » s'exclama Agar, tout excité. « Peut-être que ce rustre de Trophon, qui est venu ici à la recherche de Meline et de son laboratoire, a emporté une sorte d'orbe capable de briser les barrières, ou quelque chose comme ça.

Reviewer: ```suggestion
Le laboratoire de la Nécromancienne Meline était un lieu très imposant. Dans un coin, je pouvais voir d'énormes flacons remplis de liquides bouillonnants ; des récipients étranges pendaient à des chaînes fixées au plafond ; et devant, dans une large caverne, je pouvais voir trois Dragons Squelettes enchaînés ensemble. Sur l'une des tables jonchées de traités magiques, j'ai remarqué le journal de la nécromancienne.	Les dernières entrées étaient éloquentes :		« ... 5 février. Une autre expérience a échoué. Les bocaux contenant les âmes ont été détruits et les dragons sont morts pour de bon. Peut-être que les Dragons Noirs ne conviennent pas à cet usage. 		24 février. Mes morts-vivants ont réussi à capturer trois Dragons Verts dans la grotte voisine. Il n'y a aucune raison de se décourager, je suis sûr que je vais réussir cette fois-ci.		25 février. Une nouvelle erreur de calcul, un nouvel échec. Je vais devoir envoyer les morts-vivants à la recherche d'un nouveau dragon. De plus, l'un des fantômes m'a dit que le vieux Trophon fouinait dans les parages. C'est bien la dernière chose dont j'ai besoin. Je ne peux pas me permettre de perdre ce laboratoire aussi... » 		« Mais que se passe-t-il ici ? » m'écriai-je, surpris, en refermant le journal et en regardant autour de moi.		Agar ne semblait pas perplexe face à cet étrange laboratoire. « Ce qui se passe ici est très clair. Quelqu'un essayait de créer un Dragon Fantome. Encore une fois. Le fait établi de longue date que cela est totalement irréalisable ne semble empêcher personne d'essayer. Vous savez, c'est le même problème que certains artisans peu sensés et leurs idées saugrenues de machine à mouvement perpétuel. Sauf que leurs inventions, au pire, explosent simplement, alors que cette chose-là peut produire des fruits vraiment pourris ! » 		« Oh, je vous en prie ! À Bracada, mes collègues m'ont aussi dit que... »		Me souvenant d'un événement passé, je faillis m'offusquer des propos insolents du vieil homme, mais Kastore, qui était resté silencieux jusqu'alors, intervint dans la conversation. 		« Si nous avions moins de soucis à nous faire en ce moment, je vous parlerais du mouvement perpétuel, mes chers... érudits. Mais regardez, qu'est-ce que c'est que ça ? » Son attention fut attirée par une porte fermée dans un coin du laboratoire.		« Je parie que c'est une salle au trésor. Et scellée par magie, en plus ! » s'exclama Agar, tout excité. « Peut-être que ce rustre de Trophon, qui est venu ici à la recherche de Meline et de son laboratoire, a emporté une sorte d'orbe capable de briser les barrières, ou quelque chose comme ça.
```

Vous ne pouvez pas ouvrir le trésor ; vous devez trouver l'orbe capable de supprimer la barrière magique. 
Le trésor regorgeait d'armes en tout genre : épées magiques, lances, arcs et frondes. Il y avait même des fusils fabriqués par Burton ; aucune idée de ce que le nécromancien comptait en faire. Prenons-nous les armes et équipons-en nos forces ?

(19, 47, 1) ***Sign***
Message:
Ne nourrissez pas les wyvernes

(121, 49, 1) ***Overlord***
Name:
Kotus

(85, 32, 1) ***Seer's Hut***
Message:

> Des vers grouillaient autour de ce petit bâtiment. Le spectacle était tout à fait étrange : ils semblaient exécuter une sorte de danse complexe ; il y avait un ordre particulier et même une certaine grâce dans leurs mouvements. Ils ne prêtaient aucune attention à notre approche. « Par tout ce qui est... », murmura Agar. « Je parierais ma barbe que quelque chose ici a réussi à prendre le contrôle de cet essaim à la Matriarche ! De plus, cela a également inculqué un comportement tout à fait inhabituel à mes vers ! Regardez : ils ne montrent aucun signe d'instinct de conservation. C'est comme s'ils se moquaient de ce qui se passe autour d'eux. On dirait que leur seul désir maintenant est de ne laisser personne s'approcher de cette petite maison. Et... et ils sont... non, c'est impossible, cela ne pourrait pas arriver à des animaux qui n'ont que les rudiments les plus élémentaires de la conscience de soi... mais si je ne le savais pas, je penserais qu'ils... Ils s'exhibent ! Euh, eh bien, wow ! Je n'avais jamais prêté attention à cet aspect de mes créations auparavant. Mais à bien y réfléchir... avec leur symétrie parfaite, leurs couleurs délicates, leurs mouvements gracieux... »		« Mon ami, » s'adressa soudain Kastore à Agar, fasciné, avec autant de douceur qu'à un enfant, « serait-il possible de reporter un peu ton enthousiasme esthétique ? Penses-tu pouvoir reprendre le contrôle de ces mignonnes petites créatures ? Je te promets que dès que nous serons sortis d'ici, j'organiserai personnellement, à mes frais, un concours de beauté avec les vers, les yeux maléfiques et les troglodytes les plus adorables qui soient, et j'inviterai même Tor Anwyn en personne à présider le jury. Mais ici et maintenant, nous avons fort à faire. » « ... Eh bien, euh, ça devrait être assez facile... Il suffit de se souvenir des sorts de contrôle... »

Reviewer: ```suggestion
Des vers grouillaient autour de ce petit bâtiment. Le spectacle était tout à fait étrange : ils semblaient exécuter une sorte de danse complexe ; il y avait un ordre particulier et même une certaine grâce dans leurs mouvements. Ils ne prêtaient aucune attention à notre approche.		« Par tout ce qui est... », murmura Agar. « Je parierais ma barbe que quelque chose ici a réussi à prendre le contrôle de cet essaim provenant de la Matriarche ! De plus, cela a également inculqué un comportement tout à fait inhabituel à mes vers ! Regardez : ils ne montrent aucun signe d'instinct de conservation. C'est comme s'ils se moquaient de ce qui se passe autour d'eux. On dirait que leur seul désir maintenant est de ne laisser personne s'approcher de cette petite maison. Et... et ils sont... non, c'est impossible, cela ne pourrait pas arriver à des animaux qui n'ont que les rudiments les plus élémentaires de la conscience de soi... mais si je ne le savais pas, je penserais qu'ils... Ils s'exhibent ! Euh, eh bien, wow ! Je n'avais jamais prêté attention à cet aspect de mes créations auparavant. Mais à bien y réfléchir... avec leur symétrie parfaite, leurs couleurs délicates, leurs mouvements gracieux... »		« Mon ami, » s'adressa soudain Kastore à un Agar fasciné, avec autant de douceur que s'il s'adressait un enfant, « serait-il possible de reporter un peu votre enthousiasme esthétique ? Pensez-vous pouvoir reprendre le contrôle de ces mignonnes petites créatures ? Je vous promets que dès que nous serons sortis d'ici, j'organiserai personnellement, à mes frais, un concours de beauté avec les vers, les yeux maléfiques et les troglodytes les plus adorables qui soient, et j'inviterai même Tor Anwyn en personne à présider le jury. Mais ici et maintenant, nous avons fort à faire. »		« ... Eh bien, euh, ça devrait être assez facile... Il suffit de se souvenir des sorts de contrôle... »
```

Le récepteur avale docilement les ressources une par une, mais dès que vous cessez de les verser, il émet un hurlement offensé et recrache tout ce que vous avez mis dedans. « Matériaux insuffisants pour accorder    Déposez le montant spécifié »

> Après avoir avalé une quantité fabuleuse de ressources précieuses, la boîte émet un agréable tintement ; sur la plaque lumineuse, que Kastore appelle « écran », apparaît l'image d'un artefact, accompagnée des mots suivants : « Matériaux requis déposés. Accorder le souhait ? Pour confirmer, prononcez « Oui ». Recevoir l'artefact ?

Reviewer: ```suggestion
Après avoir avalé une quantité fabuleuse de ressources précieuses, la boîte émet un agréable tintement ; sur la plaque lumineuse, que Kastore appelle « écran », apparaît l'image d'un artefact, accompagnée des mots suivants :		« Matériaux requis déposés.		Accorder le souhait ? Pour confirmer, prononcez « Oui ».		Recevoir l'artefact ?
```

(106, 51, 1) ***Overlord***
Name:
Breus

(28, 54, 1) ***Evil Eye***
Message:

> Je progressais prudemment dans les grottes qui s'enfonçaient dans les profondeurs depuis le laboratoire d'Agar. Pendant un instant, j'eus l'impression que mes pouvoirs magiques s'évanouissaient... mais ce ne fut que momentané. Mes pensées furent interrompues par des bruits étranges. Des yeux maléfiques apparurent devant moi ! Évidemment. J'aurais dû deviner qu'Agar laisserait ses « bêtes » préférées garder son laboratoire !

Reviewer: ```suggestion
Je progressais prudemment dans les grottes qui s'enfonçaient dans les profondeurs depuis le laboratoire d'Agar. Pendant un instant, j'eus l'impression que mes pouvoirs magiques s'évanouissaient... mais ce ne fut que momentané. Mes pensées furent interrompues par des bruits étranges. Des Yeux Maléfiques apparurent devant moi ! Évidemment. J'aurais dû deviner qu'Agar laisserait ses « bêtes » préférées garder son laboratoire !
```

(134, 63, 1) ***Mercenary***
Name:
Henrietta

(135, 65, 1) ***Olgoi-Khorkhoi***
Message:

> Nous courions dans le tunnel depuis des heures. Je ne saurais dire combien de temps s'était écoulé, mais cela faisait un bon moment que j'avais commencé à trouver des inscriptions sur les murs et les colonnes, dans une langue que j'avais inventée moi-même et enseignée à Henrietta autrefois. Le labyrinthe de grottes et de catacombes semblait sans fin, mais nous étions sans aucun doute sur la bonne piste. Ma seule inquiétude était d'arriver trop tard... Il était impossible que l'équipe des hobbits ait suffisamment de provisions pour tenir tous ces mois, ou peu importe le temps que nous avions passé sous terre. Je ne savais plus si je pouvais me fier à ma perception du temps. Kastore me fit signe. Certaines parois de la grotte semblaient avoir fondu, comme si elles avaient été touchées par un rayon de feu. Puis nous entendîmes un bourdonnement étrange ; une lumière clignota au-dessus de nous, et tout se figea à nouveau. Un instant plus tard, une odeur de viande brûlée nous parvint. Kastore dégaina son arme, tandis que je préparais mes sorts. Nous nous précipitâmes dans le passage d'où venait de jaillir cette lumière aveuglante. Plus nous nous approchions de la source de cette horrible odeur, plus nous trouvions de coquilles chitineuses carbonisées de vers. Finalement, j'aperçus Henrietta. La jeune fille était émaciée, le visage sale et griffé, mais elle était vivante ! Elle tenait dans ses mains quelque chose qui ressemblait à une boule de pierre et l'agitait dans tous les sens.		« Frederick ! Attention ! Il y en a des centaines ici ! Cette chose, elle... elle les brûle ! » Je ne comprenais pas très bien de quoi elle parlait. La boule de pierre ? Mais comment est-ce possible... Le mur à notre droite s'est effondré. J'ai vu les gueules béantes des vers — ils étaient sur le point de se précipiter vers nous. Soudain, l'espace fut rempli d'un bourdonnement mécanique. Je fus aveuglé pendant une seconde, et lorsque ma vision revint, les vers se tordaient dans les flammes, et la boule de pierre d'Henrietta brillait d'une lueur similaire à celle du flash, mais plus douce. « Frederick, ne reste pas planté là comme un idiot ! Regarde autour de toi ! » me siffla Kastore, mais il était trop tard. Le sol sous nos pieds s'est ouvert et s'est effondré ; nous sommes tombés à un niveau encore plus bas des grottes. C'est là que nous avons rencontré la Matriarche. C'était son repaire ; il grouillait de larves et de vers adultes, et il y avait encore plus de ses œufs hideux. « Eh bien... on dirait que nous les avons trouvées toutes les deux : Henrietta et la Matriarche », ai-je murmuré en me préparant au combat.

Reviewer: ```suggestion
Nous courions dans le tunnel depuis des heures. Je ne saurais dire combien de temps s'était écoulé, mais cela faisait un bon moment que j'avais commencé à trouver des inscriptions sur les murs et les colonnes, dans une langue que j'avais inventée moi-même et enseignée à Henrietta autrefois. Le labyrinthe de grottes et de catacombes semblait sans fin, mais nous étions sans aucun doute sur la bonne piste. Ma seule inquiétude était d'arriver trop tard... Il était impossible que l'équipe des hobbits ait suffisamment de provisions pour tenir tous ces mois, ou peu importe le temps que nous avions passé sous terre. Je ne savais plus si je pouvais me fier à ma perception du temps.		Kastore me fit signe. Certaines parois de la grotte semblaient avoir fondu, comme si elles avaient été touchées par un rayon de feu. Puis nous entendîmes un bourdonnement étrange ; une lumière clignota au-dessus de nous, et tout se figea à nouveau. Un instant plus tard, une odeur de viande brûlée nous parvint. Kastore dégaina son arme, tandis que je préparais mes sorts. Nous nous précipitâmes dans le passage d'où venait de jaillir cette lumière aveuglante. Plus nous nous approchions de la source de cette horrible odeur, plus nous trouvions de coquilles chitineuses carbonisées de vers. Finalement, j'aperçus Henrietta. La jeune fille était émaciée, le visage sale et griffé, mais elle était vivante ! Elle tenait dans ses mains quelque chose qui ressemblait à une boule de pierre et l'agitait dans tous les sens.		« Frederick ! Attention ! Il y en a des centaines ici ! Cette chose, elle... elle les brûle ! »		Je ne comprenais pas très bien de quoi elle parlait. La boule de pierre ? Mais comment est-ce possible... Le mur à notre droite s'est effondré.		J'ai vu les gueules béantes des vers — ils étaient sur le point de se précipiter vers nous. Soudain, l'espace fut rempli d'un bourdonnement mécanique. Je fus aveuglé pendant une seconde, et lorsque ma vision revint, les vers se tordaient dans les flammes, et la boule de pierre d'Henrietta brillait d'une lueur similaire à celle du flash, mais plus douce.		« Frederick, ne reste pas planté là comme un idiot ! Regardez autour de vous ! » me siffla Kastore, mais il était trop tard. Le sol sous nos pieds s'est ouvert et s'est effondré ; nous sommes tombés à un niveau encore plus bas des grottes. C'est là que nous avons rencontré la Matriarche. C'était son repaire ; il grouillait de larves et de vers adultes, et il y avait encore plus de ses œufs hideux.		« Eh bien... on dirait que nous les avons trouvées toutes les deux : Henrietta et la Matriarche », ai-je murmuré en me préparant au combat.
```

(104, 91, 1) ***Olgoi-Khorkhoi***
Message:
Nous avons traversé la tombe avec prudence, à la recherche de traces hobbits ou d'humains. Un bruit s'est fait entendre devant nous, un bruit que je connaissais bien. Il y avait des vers des sables. Je compris alors clairement ce qui s'était passé ici : les vers avaient creusé des galeries qui avaient provoqué l'effondrement de la tombe. À l'endroit où Henrietta et son expédition étaient récemment passées, nous ne trouvâmes que des tas de gravats : des pierres, des colonnes et des stèles anciennes brisées, toutes couvertes d'inscriptions inconnues. Il fallait trouver un autre chemin...

> 

Reviewer: Espace à retiré

Cette pièce est protégée par une barrière magique, créée peut-être par la puissante nécromancienne Ioke. Tant qu'elle maintient cette barrière, il n'y a aucun moyen de passer.
Dans la pièce, vous trouvez un orbe bleu caché derrière une barrière de verre. Voulez-vous la briser ?

(4, 117, 1) ***Seer's Hut***
Message:
Cette pièce est protégée par une barrière magique, créée peut-être par la puissante nécromancienne Ioke. Tant qu'elle maintient cette barrière, il n'y a aucun moyen de passer.
Cette pièce est protégée par une barrière magique, créée peut-être par la puissante nécromancienne Ioke. Tant qu'elle maintient cette barrière, il n'y a aucun moyen de passer.
Dans la pièce, vous trouvez un orbe blanc caché derrière une barrière de verre. Voulez-vous la briser ?

(14, 84, 1) ***Quest Guard***
Message:
La porte menant aux profondeurs de la tombe est fermée à clé, et vous ne parvenez pas à l'ouvrir. Cependant, en regardant de plus près, vous remarquez une serrure inhabituelle en forme d'ankh.
Cette partie de la tombe vous est encore inaccessible. Vous devez trouver un autre moyen. 

> À l'aide du pendentif de vie, vous pouvez ouvrir la porte, mais celui-ci perdra alors son pouvoir.     Voulez-vous entrer ?

Reviewer: ```suggestion
À l'aide du Pendentif de Vie, vous pouvez ouvrir la porte, mais celui-ci perdra alors son pouvoir.     Voulez-vous entrer ?
```

(140, 105, 1) ***Quest Guard***
Message:

> Cette pièce est protégée par une barrière magique, probablement créée par la puissante élémentaliste Yrkula. Tant qu'elle maintient cette barrière, il est impossible de passer.

Reviewer: ```suggestion
Cette pièce est protégée par une barrière magique, probablement créée par la puissante Conjureureuse Yrkula. Tant qu'elle maintient cette barrière, il est impossible de passer.
```

> Cette pièce est protégée par une barrière magique, probablement créée par la puissante élémentaliste Yrkula. Tant qu'elle maintient cette barrière, il est impossible de passer.

Reviewer: ```suggestion
Cette pièce est protégée par une barrière magique, probablement créée par la puissante Conjureureuse Yrkula. Tant qu'elle maintient cette barrière, il est impossible de passer.
```

Maintenant qu'Yrkula a été vaincue, la barrière magique a disparu. Voulez-vous passer ?

(58, 64, 1) ***Quest Guard***
Message:

> La première entrée du laboratoire d'Agar était bloquée par une porte faite d'un métal étrange que vous n'aviez jamais vu auparavant. Après plusieurs tentatives pour forcer ou faire fondre la porte, il est devenu évident que vous perdiez votre temps. Tu as repéré un trou de serrure, mais il était clair que la porte ne pouvait être ouverte que de l'intérieur.     Déçu, tu examines la porte une dernière fois et t'apprêtes à partir lorsque tu remarques soudain des empreintes d'ogres et d'orques sur le sol de la grotte. Certaines mènent manifestement au laboratoire. Peut-être qu'Agar n'est pas le seul à pouvoir y entrer ? 

Reviewer: ```suggestion
La première entrée du laboratoire d'Agar était bloquée par une porte faite d'un métal étrange que vous n'aviez jamais vu auparavant. Après plusieurs tentatives pour forcer ou faire fondre la porte, il est devenu évident que vous perdiez votre temps. Vous avez repéré un trou de serrure, mais il était clair que la porte ne pouvait être ouverte que de l'intérieur.     Déçu, vous examinez la porte une dernière fois et vous apprêtez à partir lorsque vous remarquez soudain des empreintes d'ogres et d'orques sur le sol de la grotte. Certaines mènent manifestement au laboratoire. Peut-être qu'Agar n'est pas le seul à pouvoir y entrer ? 
```

Les nouvelles tentatives pour forcer la porte ont échoué. Vous devez trouver la clé... 
Vous avez trouvé un objet étrange dans la forteresse montagneuse du barbare Akratos. De retour au laboratoire, vous réalisez qu'il s'agit de la clé du repaire d'Agar ! Souhaitez-vous entrer ?

(115, 31, 1) ***Seer's Hut***
Message:

> Kastore et moi contemplions les murs de la Salle des Illusions avec curiosité. « Quel spectacle merveilleux », dis-je lentement. « Beaucoup de mages peuvent créer des illusions, mais je n'en ai jamais vu de pareilles auparavant. Habituellement, elles sont facilement dissipées d'un coup d'épée, mais celles-ci... » Je m'interrompis lorsque j'aperçus un génie devant moi.		« Que regardez-vous ? » rugit-il avec indignation. « Je ne suis pas une illusion. Cela peut paraître étrange, mais je veux sortir d'ici. J'en ai assez de cet endroit, mais les gardiens des tombes sont assez puissants pour empêcher tout génie de quitter leur chambre. » « Les gardiens des tombes ? » « Ils prennent différentes formes, et vous en avez peut-être déjà rencontré certains. Je ne sais pas qui garde la Salle des Illusions, mais je pense que ce sont les dragons féériques. Aide-moi à m'occuper d'eux, et je t'apprendrai la magie de combat. »

Reviewer: ```suggestion
Kastore et moi contemplions les murs de la Salle des Illusions avec curiosité.		« Quel spectacle merveilleux », dis-je lentement. « Beaucoup de mages peuvent créer des illusions, mais je n'en ai jamais vu de pareilles auparavant. Habituellement, elles sont facilement dissipées d'un coup d'épée, mais celles-ci... »		Je m'interrompis lorsque j'aperçus un Génie devant moi.		« Que regardez-vous ? » rugit-il avec indignation. « Je ne suis pas une illusion. Cela peut paraître étrange, mais je veux sortir d'ici. J'en ai assez de cet endroit, mais les gardiens des tombes sont assez puissants pour empêcher tout génie de quitter leur chambre. »		« Les gardiens des tombes ? »		« Ils prennent différentes formes, et vous en avez peut-être déjà rencontré certains. Je ne sais pas qui garde la Salle des Illusions, mais je pense que ce sont les Dragons Féeriques. Aidez-moi à m'occuper d'eux, et je vous apprendrai la magie de combat. »
```

Je suis toujours esclave entre ces murs. Les dragons féériques sont dans les parages.

> Pour la première fois en mille cinq cents ans, j'ai goûté à la liberté, et j'adore cette sensation ! Veux-tu commencer ton entraînement maintenant ?

Reviewer: ```suggestion
Pour la première fois en mille cinq cents ans, j'ai goûté à la liberté, et j'adore cette sensation !		Voulez-vous commencer votre entraînement maintenant ?
```

Malheureusement, vous n'avez pas les artefacts que je recherche.
Cette cape et ce manteau sont exactement ce dont j'avais besoin ! Je n'ai pas d'or pour vous payer, mais je peux vous donner une bague enchantée qui supprime toute magie de résurrection. Qui sait quand elle pourrait vous être utile ?

(107, 106, 1) ***Efreet Sultan***
Message:

> Dès que je me suis approché du portail, plus d'une douzaine de tourbillons de feu, des effrits, sont apparus de nulle part. J'ai ordonné à mes hommes de préparer leurs armes.

Reviewer: ```suggestion
Dès que je me suis approché du portail, plus d'une douzaine de tourbillons de feu, des Effrits, sont apparus de nulle part. J'ai ordonné à mes hommes de préparer leurs armes.
```

(41, 22, 1) ***Event***
Message:

> Les cavernes inondées réapparurent, et il semblait que nous allions devoir reprendre le bateau. Agar nous avait prévenus : il avait lui-même tenté à plusieurs reprises d'atteindre l'autre rive de ce lac souterrain, par magie et en bateau, mais le lac était truffé de rochers sous-marins, et l'autre rive était trop éloignée pour être atteinte par un sortilège. Peut-être pourrions-nous voir ce que nous avions besoin de voir en marchant sur l'eau autour de notre navire ?

Reviewer: ```suggestion
Les cavernes inondées réapparurent, et il semblait que nous allions devoir reprendre le bateau. Agar nous avait prévenus : il avait lui-même tenté à plusieurs reprises d'atteindre l'autre rive de ce lac souterrain, par magie et en bateau, mais le lac était truffé de rochers sous-marins, et l'autre rive était trop éloignée pour être atteinte par un sortilège.		Peut-être pourrions-nous voir ce que nous avions besoin de voir en marchant sur l'eau autour de notre navire ?
```

(159, 176, 1) ***Seer's Hut***
Message:
Vous vous battez bien, mais vous préférerez peut-être nos automates à ceux que vous possédez. Apportez-nous 10 de mercure et 20 000 pièces d'or, et nous vous fournirons volontiers de nouveaux modèles.
Malheureusement, cela ne suffit pas.
C'est parfait !
Apportez-nous 2 automates et nous les remplacerons par de meilleurs.
Malheureusement, nous ne pouvons pas encore vous aider. Revenez plus tard !
Voulez-vous faire l'échange maintenant ?

(149, 177, 1) ***Seer's Hut***
Message:
Tu te bats bien, mais tu préféreras peut-être notre équipement à celui que tu possèdes. Apporte-nous 10 de soufre et 20 000 pièces d'or, et nous te fournirons volontiers.
Malheureusement, cela ne suffit pas.
C'est génial !
Apporte-nous 6 mécaniciens et nous les équiperons de nouveaux outils.

> Malheureusement, nous ne pouvons pas encore t'aider. Reviens plus tard !

Reviewer: ```suggestion
Malheureusement, nous ne pouvons pas encore vous aider. Revenez plus tard !
```

Voulez-vous faire un échange maintenant ?

(161, 177, 1) ***Seer's Hut***
Message:

> Tu te bats bien, mais tu préféreras peut-être nos fusils à ceux que tu possèdes. Apporte-nous 10 gemmes et 20 000 pièces d'or, et nous te fournirons volontiers de nouvelles armes.

Reviewer: ```suggestion
Vous vous battez bien, mais vous préférerez peut-être nos fusils à ceux que vous possédez. Apportez-nous 10 gemmes et 20 000 pièces d'or, et nous vous fournirons volontiers de nouvelles armes.
```

Malheureusement, cela ne suffit pas.
C'est génial !
Apporte-nous 1 pistolero et nous lui fournirons un nouveau pistolet.
Malheureusement, nous ne pouvons pas encore t'aider. Reviens plus tard !
Voulez-vous faire un échange maintenant ?

(152, 176, 1) ***Seer's Hut***
Message:

> Tu te bats bien, mais tu préféreras peut-être nos bêtes à celles que tu possèdes. Apporte-nous 10 cristaux et 20 000 pièces d'or, et nous te fournirons volontiers de meilleures bêtes.

Reviewer: ```suggestion
Vous vous battez bien, mais vous préférerez peut-être nos bêtes à celles que vous possédez. Apportez-nous 10 cristaux et 20 000 pièces d'or, et nous vous fournirons volontiers de meilleures bêtes.
```

Malheureusement, cela ne suffit pas.
C'est génial !
Apporte-nous 4 glyptodons et nous les remplacerons par de plus puissants.

> Malheureusement, nous ne pouvons pas encore t'aider. Reviens plus tard !

Reviewer: ```suggestion
Malheureusement, nous ne pouvons pas encore vous aider. Revenez plus tard !
```

(87, 68, 1) ***Event***
Message:

> Une étrange sensation envahit ton esprit. Pendant un instant, tu as eu l'impression que le temps lui-même s'était arrêté...

Reviewer: ```suggestion
Une étrange sensation envahit votre esprit. Pendant un instant, vous avez eu l'impression que le temps lui-même s'était arrêté...
```

(51, 2, 1) ***Elementalist***
Name:
Fulghea

(106, 106, 1) ***Quest Guard***
Message:
Ce portail à sens unique mène clairement au cœur de la tombe, mais il est peu probable qu'Henrietta et son expédition aient emprunté ce chemin.
Nous devons découvrir où l'expédition d'Henrietta s'est rendue.
Voulez-vous vous diriger vers le portail ?

(80, 107, 1) ***Quest Guard***
Message:
Le chemin menant à cette partie du tombeau a été scellé par magie par les maîtres génies. Tant qu'ils sont là, il est impossible d'ouvrir cette porte.
Le chemin menant à cette partie du tombeau a été scellé par magie par les maîtres génies. Tant qu'ils sont là, il est impossible d'ouvrir cette porte.

> Les maîtres génies ont été vaincus, et le passage vers cette partie du tombeau est désormais ouvert. Voulez-vous passer ?

Reviewer: ```suggestion
Les Génies ont été vaincus, et le passage vers cette partie du tombeau est désormais ouvert.	Voulez-vous passer ?
```

(160, 135, 1) ***Pandora's Box***
Message:

(160, 136, 1) ***Quest Guard***
Message:
Le chemin menant à cette partie du tombeau a été scellé par magie par des gardes après l'intrusion du dragon azur. Peut-être que si vous chassez le monstre, la barrière sera levée.
Le chemin menant à cette partie du tombeau a été scellé par magie par des gardes après l'intrusion du dragon azur. Peut-être que si vous chassez le monstre, la barrière sera levée.
Le dragon n'est plus et le passage vers cette partie du tombeau est ouvert. Voulez-vous passer ?

(164, 135, 1) ***Pandora's Box***
Message:

(164, 136, 1) ***Quest Guard***
Message:

> Le chemin menant à cette partie du tombeau a été scellé par magie par des gardes après l'intrusion des dragons fantômes. Peut-être que si vous chassez les monstres, la barrière sera levée.

Reviewer: ```suggestion
Le chemin menant à cette partie du tombeau a été scellé par magie par des gardes après l'intrusion des Dragons Fantômes. Peut-être que si vous chassez les monstres, la barrière sera levée.
```

Le chemin menant à cette partie du tombeau a été scellé par magie par des gardes après l'intrusion des dragons fantômes. Peut-être que si vous chassez les monstres, la barrière sera levée.
Les dragons ont disparu et le passage vers cette partie du tombeau est ouvert. Voulez-vous passer ?

(168, 136, 1) ***Quest Guard***
Message:

> Le chemin menant à cette partie du tombeau a été scellé par magie par les maîtres génies. Tant qu'ils sont là, il est impossible d'ouvrir cette porte.

Reviewer: ```suggestion
Le chemin menant à cette partie du tombeau a été scellé par magie par les Génies. Tant qu'ils sont là, il est impossible d'ouvrir cette porte.
```

> Le chemin menant à cette partie du tombeau a été scellé par magie par les maîtres génies. Tant qu'ils sont là, il est impossible d'ouvrir cette porte.

Reviewer: ```suggestion
Le chemin menant à cette partie du tombeau a été scellé par magie par les Génies. Tant qu'ils sont là, il est impossible d'ouvrir cette porte.
```

(172, 136, 1) ***Quest Guard***
Message:
Le passage vers cette partie de la tombe est protégé par une barrière magique, créée peut-être par la puissante nécromancienne Ioke. Tant qu'elle maintient cette barrière, il n'y a aucun moyen de passer.

> Le passage vers cette partie de la tombe est protégé par une barrière magique, créée peut-être par la puissante nécromancienne Ioke. Tant qu'elle maintient cette barrière, il n'y a aucun moyen de passer.

Reviewer: ```suggestion
Le passage vers cette partie de la tombe est protégé par une barrière magique, créée peut-être par la puissante Nécromancienne Ioke. Tant qu'elle maintient cette barrière, il n'y a aucun moyen de passer.
```

Maintenant que Ioke a été vaincue, la barrière magique a disparu. Voulez-vous passer ?

(176, 136, 1) ***Quest Guard***
Message:
Un groupe de génies bloque l'entrée de cette partie de la tombe. Ils acceptent de vous laisser entrer si vous leur apportez l'artefact perdu : le Pendentif du Courage.
Un groupe de génies bloque l'entrée de cette partie de la tombe. Ils acceptent de vous laisser entrer si vous leur apportez l'artefact perdu : le Pendentif du Courage.

> Les génies remarquent que vous avez le pendentif et sont prêts à vous laisser entrer. Voulez-vous passer ?

Reviewer: ```suggestion
Les Djinns remarquent que vous avez le pendentif et sont prêts à vous laisser entrer.	Voulez-vous passer ?
```

(176, 135, 1) ***Pandora's Box***
Message:

(160, 138, 1) ***Quest Guard***
Message:
Un groupe de génies bloque l'entrée de cette partie de la tombe. Ils acceptent de vous laisser entrer si vous leur apportez l'artefact perdu : le Pendentif de Pureté
Un groupe de génies bloque l'entrée de cette partie de la tombe. Ils acceptent de vous laisser entrer si vous leur apportez l'artefact perdu : le Pendentif de Pureté.

> Les génies remarquent que vous avez le pendentif et sont prêts à vous laisser entrer. Voulez-vous passer ?

Reviewer: ```suggestion
Les Djinns remarquent que vous avez le pendentif et sont prêts à vous laisser entrer.	Voulez-vous passer ?
```
